python/src/trezorlib/cli/trezorctl.py

```diff
@@ -76,9 +76,10 @@
     "xlm": stellar.cli,
     "xtz": tezos.cli,
     # firmware-update aliases:
-    "update-firmware": firmware.firmware_update,
-    "upgrade-firmware": firmware.firmware_update,
-    "firmware-upgrade": firmware.firmware_update,
+    "update-firmware": firmware.update,
+    "upgrade-firmware": firmware.update,
+    "firmware-upgrade": firmware.update,
+    "firmware-update": firmware.update,
```

Comment:
while you're touching this, please also add a `fw` alias for the `firmware` subcommand :)

Reply:
Added in 2ae4